WcaOnRails/app/views/competitions/_index_form.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="form-group">
```

Comment:
Wow, those do look awful in firefox, I can see why you don't like them, lol.

Your use of `label_tag` looks great to me!

Maybe a solution for lining up the buttons would be to add a `<label>`, just like the rest of the things? The `<label>` probably doesn't need to have anything in it, so you might need to put a nonbreaking space or something gross (`&nbsp;`)